mods/voidmulti.sc2mod/base.sc2data/gamedata/unitdata.xml

```diff
@@ -569,10 +739,11 @@
         <Food value="6"/>
         <Collide index="Locust" value="1"/>
         <Collide index="Phased" value="1"/>
-        <SubgroupPriority value="32"/>
+        <SubgroupPriority value="28"/>
         <CardLayouts index="0">
             <LayoutButtons index="10" removed="1"/>
         </CardLayouts>
+        <Sight value="12"/>
```

Comment:
Hatchery sight range increased from 10 to 12 to reach to the end of its own creep spread